crates/driver/src/infra/pod/config.rs

```diff
@@ -0,0 +1,9 @@
+use {crate::domain::eth, serde::Deserialize, serde_with::serde_as, url::Url};
+
+#[serde_as]
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct Config {
+    pub endpoint: Url,
+    pub auction_contract_address: eth::H160,
```

Comment:
How do you envision serving all the different chains on `pod`? I mean would it make the most sense to have 2 pod contract per supported chain (1 for staging and 1 for prod) or would it make more sense to tag submitted solution with their chain id and env? I think it doesn't matter too much for our code but perhaps one option scales better on pod.